### ISSUE TITLE
Master.sql should be embeded as resource. 

### ISSUE THREAD

**Issue by chlupac** [oqtane/oqtane.framework#297]
Master.sql is not part of publish and after site publishing is not copied to correct place.

dbUp library is capable to do whole database installation process including database creation and variable substitution. I have tested:

```c#
                
            EnsureDatabase.For.SqlDatabase(connectionString);
            
            var dbUpgradeConfig = DeployChanges
                .To
                .SqlDatabase(connectionString)
                .WithVariable("ConnectionString",connectionString)
                .WithVariable("Alias", silentInstall.Alias)
                .WithScriptsEmbeddedInAssembly(Assembly.GetExecutingAssembly(),s => s.StartsWith("Master")) 
                .WithScriptsEmbeddedInAssembly(Assembly.GetExecutingAssembly(),s => !s.StartsWith("Master"));
            var dbUpgrade = dbUpgradeConfig.Build();
            if (dbUpgrade.IsUpgradeRequired())
            {
                var result = dbUpgrade.PerformUpgrade();
                if (!result.Successful)
                {
                    Console.WriteLine(result.Error.Message);
                }
                else
                {
                        UpdateAppSettings(connectionString);
                }
            }
```

**Comment by sbwalker** [comment]
If you have tested this thoroughly and it works then I think this is a good improvement,.

**Comment by chlupac** [comment]
Solved in #306
